ui/pages/confirmations/components/simulation-details/simulation-details.tsx

```diff
@@ -60,13 +55,9 @@ const ErrorContent: React.FC<{ error: SimulationError }> = ({ error }) => {
   const t = useI18nContext();
 
   function getMessage() {
-    if (
-      error.isReverted ||
-      error.message?.includes('insufficient funds for gas')
-    ) {
-      return t('simulationDetailsTransactionReverted');
-    }
-    return t('simulationDetailsFailed');
+    return error.code === SimulationErrorCode.Reverted
+      ? t('simulationDetailsTransactionReverted')
+      : t('simulationDetailsFailed');
   }
```

Comment:
```suggestion
  const message = error.code === SimulationErrorCode.Reverted
      ? t('simulationDetailsTransactionReverted')
      : t('simulationDetailsFailed');
```

Reply:
Not sure what you mean here?

This previously returned so I'm doing the same but with a ternary.

Reply:
Hey, my bad for not providing more context. Just a small nit - with the ternary operator, we just have a single expression and can replace the getMessage function with `const message = ...`. It's very minor.